doc/openfortivpn.1

```diff
@@ -115,14 +126,24 @@ username = foo
 .br
 password = bar
 .br
+user-cert = /etc/openfortivpn/user-cert.pem
+.br
+user-key =/etc/openfortivpn/user-key.pem
+.br
```

Comment:
This section is just an example, I prefer to keep it clear by just having a few options (not all).